### ISSUE TITLE
Cumulative ignore reader macros

### ISSUE THREAD

**Issue by helins** [Convex-Dev/convex#264]
Not top priority but Clojure allows to chain several `#_`. For instance, the following fails with the Convex Reader while the Clojure Reader would effectively ignore both forms:

```clojure
#_ #_ :foo :bar
```

Sometimes useful.

**Comment by mikera** [comment]
Hmmm the intention is to ignore both?

**Comment by helins** [comment]
Yes, you can't "ignore ignoring", they have a cumulative effect. Since there are 2 `#_` in the example, the next 2 forms afterwards should be ignored.

Subtle touch, granted, but sometimes useful during dev. Allows for quickly ignoring forms in a row.

**Comment by brittleboye** [comment]
Implemented on develop in 1164c5d99. Consecutive #_ reader discard markers now each consume one following form, with direct ANTLR coverage, compiler integration coverage and malformed-input tests. The full clean reactor build passed.
